docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -2,25 +2,26 @@
 
 When you configure the Digger Installer to use proxy, it will automatically reconfigure the following components:
```

Comment:
I would suggest not using the work `reconfigure` here and use configure instead. It's best for the user to provide this variables in advance as suggested in the installer guide.

Reply:
@laurafitzgerald  isnt' the installer idempotent, 


If you use a HTTP proxy and follow this procedure, running the Ansible playbook will configure or reconfigure the following components:

Reply:
@laurafitzgerald My pov is that the components have already been configured, hence "re"-configuration

Reply:
@finp yes the installer is idempotent but we would still rather the user complete all the configuration before the first installation.  Your suggested replacement above sounds good :+1:

Reply:
@pmacko1 if this is the first run of the installation then they would not have already been configured. I think the suggestion above made by @finp covers it.
The proxy is used during the installation if there are any resources that need to access which are outside the customers network. e.g. docker images.
to your above suggestion it would be better to say
`The installer will automatically configure the following components to use the proxy where outbound internet access is required`

Reply:
@laurafitzgerald  I'm getting out of the loop, sorry :). Digger is implicitly behind proxy?
Adding `The installer will automatically configure the following components to use the proxy where outbound internet access is required`

Reply:
resolved